app/services/to_collection_form/roundtrip_validator.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module ToCollectionForm
```

Comment:
Is this RoundtripValidator substantively different than the validator for the work form?

Reply:
Not substantially, but if I remove source_id from collection that would be one difference that would warrant keeping them separate I think.